frontend/src/pages/AccountSend.vue

```diff
@@ -15,10 +15,40 @@
     <!-- Send form -->
     <q-form v-else @submit="onFormSubmit" class="form" ref="sendFormRef">
       <!-- Identifier -->
-      <div v-if="!advancedMode">Recipient's ENS or CNS name</div>
+      <div v-if="!advancedMode">Recipient's ENS name or address</div>
       <div v-else>Recipient's ENS name, CNS name, transaction hash, address, or uncompressed public key</div>
       <base-input v-model="recipientId" :disable="isSending" placeholder="vitalik.eth" lazy-rules :rules="isValidId" />
 
+      <!-- Identifier, advanced mode tooltip -->
+      <q-checkbox
+        v-if="advancedMode"
+        v-model="useNormalPubKey"
+        class="text-caption q-pt-sm q-pb-lg"
+        dense
+        :style="!recipientId || isValidId(recipientId) === true ? 'margin-top:-2em' : ''"
+      >
+        Send using recipient's standard public key
+        <base-tooltip icon="fas fa-question-circle">
```

Comment:
The copy in this tooltip is thorough, but I'm afraid no one will ever read it.

<img width="527" alt="Screen Shot 2021-10-05 at 7 29 49 AM" src="https://user-images.githubusercontent.com/1479802/136074772-aba16a41-3311-4bad-bb22-5a702d85e712.png">

What do you think of going with something like: "When checked, the public key used will be the one underlying the user's Ethereum address. The receiver will have to manipulate their account's private key to withdrawal the funds. **Don't use this feature unless you know what you're doing**. Learn More.

Also, I wonder if the checkbox should simply be checked and disabled (i.e. greyed out and not clickable) when the user enters a tx hash or public key? Alternatively, I wonder if we should just remove that feature even in advanced mode?

Finally, if I tap the tooltip icon on mobile, which is required to show the contents, it toggles the check mark. It should not do this.

Reply:
Yea we can shorten the copy to your version.

> Also, I wonder if the checkbox should simply be checked and disabled (i.e. greyed out and not clickable) when the user enters a tx hash or public key? 

 I agree it should be disabled when user enters a public key, but entering a transaction hash raises a good question—when a transaction hash is entered, we just derive the public key directly from the transaction parameters, but alternatively we could take the from address and look it up in the registry if this box is unchecked, so should we? I learn towards no, because that seems like a useless use case, i.e. transaction hashes _always_ use the standard pubkey of the sender

> Alternatively, I wonder if we should just remove that feature even in advanced mode?

I have a strong preference towards keeping this feature, because we want the "anonymity set" of recipients to be as large as possible. Removing this makes is significantly more likely that only users in the StealthKeyRegistry are using Umbra, which shrinks the anonymity set

> Finally, if I tap the tooltip icon on mobile, which is required to show the contents, it toggles the check mark. It should not do this.

Fixed in 9ce8cbe